tests/lcs/core/engine/globalproperties.livecodescript

```diff
@@ -33,3 +33,11 @@ on TestGetSetSecurityPermissions
    set the securityPermissions to tNewPermissions
    TestAssert "Getting new securityPermissions correctly", the securityPermissions is tNewPermissions
 end TestGetSetSecurityPermissions
+
+on TestDeleteGraphicRefToGlobals
```

Comment:
I'm not sure the name of the test here accurately reflects what it is testing...